src/main/PDF/PdfMongo.java

```diff
@@ -100,15 +157,29 @@ public static JSONObject getAllFiles(
   private static JSONArray mongodbGetAllFiles(Bson filter, PDFType pdfType, MongoDatabase db) {
     JSONArray files = new JSONArray();
     GridFSBucket gridBucket = GridFSBuckets.create(db, pdfType.toString());
-    for (GridFSFile grid_out : gridBucket.find(filter)) {
-      assert grid_out.getMetadata() != null;
-      files.put(
-          new JSONObject()
-              .put("filename", grid_out.getFilename())
-              .put("uploader", grid_out.getMetadata().getString("uploader"))
-              .put("organizationName", grid_out.getMetadata().getString("organizationName"))
-              .put("id", grid_out.getId().asObjectId().getValue().toString())
-              .put("uploadDate", grid_out.getUploadDate().toString()));
+    if (pdfType.equals(PDFType.FORM)) {
+      for (GridFSFile grid_out : gridBucket.find(filter)) {
+        assert grid_out.getMetadata() != null;
+        files.put(
```

Comment:
can we just create a JSONObject with all the put fields that are the same, and then do an if else statement/case switch statement about the PDFType and add the difference?

Reply:
Done